### DISCUSSION TITLE
A way to deprecate members

### DISCUSSION THREAD

**Discussion by mwalcott3** [stephenberry/glaze#244] [Ideas]
A glz::deprecated like glz::hide may be useful when using this for config files to deprecate config options and direct people to what they should use instead kind of like [[deprecated]].
I have no need for such a thing right now but just throwing this out here since it popped in my head. But generally YAGNI

**Comment by stephenberry** [comment]
JSON Schema has a deprecated tag.

**Comment by stephenberry** [comment]
JSON schema support is quite powerful now and allows users to mark deprecated keys. I'm considering this complete. Though, more specific enhancements may be brought up in the future.
